frontend/src/pages/auth/signin/SignIn.tsx

```diff
@@ -1,11 +1,52 @@
 import React from 'react';
-import {Titles} from "../../../constansts/titles";
+import {Box, Button} from "@mui/material";
+import "../../../common/styles/button.css";
+import {useAppDispatch} from "../../../stores/redux/hooks";
+import {signIn} from "../../../stores/redux/loggedUser/loggedUserSlice";
+import {getUserService} from "../../../services/user";
+import {Buttons} from "../../../common/constansts/buttons";
+import {logInService} from "../../../services/auth";
+import {User} from "../../../models/user";
+import {UserType} from "../../../types/UserType";
+import {ILoggedUser} from "../../../types/loggedUser";
 
 const SignIn: React.FC = () => {
+    const dispatch = useAppDispatch()
+
+    const decodeToken = (token: string) => {
+        return 0;
+    }
+
+    const createLoggedUser = (token: string, user: User): ILoggedUser => {
+        return {
+            id: user.id,
+            email: user.email,
+            username: user.name,
+            userType: UserType.Customer,
+            token: token
+        };
+    }
+
+    const handleSignIn = async () => {
+        const token = await logInService({email: "user@mail.com", password: "password"});
+        const userId = decodeToken(token);
+        const user = await getUserService(userId);
```

Comment:
```suggestion
        const user = await getUserService(decodedToken.userId, decodedToken.userType);
```
getUserService will handle based on user type where to send request. Or you have to right if/else and run dedicated service getClientService or getMerchantService